tools/swift-inspect/CMakeLists.txt

```
@@ -27,7 +27,19 @@ if(WIN32)
     Sources/SwiftInspectClient/SwiftInspectClient.cpp)
   target_link_libraries(SwiftInspectClient PRIVATE
     SwiftInspectClientInterface)
-elseif(LINUX)
+endif()
+
+if (ANDROID)
+  add_library(AndroidCLib STATIC
+    Sources/AndroidCLib/heap.c)
+  target_include_directories(AndroidCLib PUBLIC
+    Sources/AndroidCLib/include)
+  target_link_options(AndroidCLib PUBLIC
+    "-Xlinker" "-T${CMAKE_SOURCE_DIR}/Sources/AndroidCLib/section-text-remote.ld")
```

Please use `SHELL:` to escape these, or better yet, just rewrite this as:

```suggestion
    "LINKER:-T${CMAKE_SOURCE_DIR}/Sources/AndroidCLib/section-text-remote.ld")
```

However, I think that there might be a better solution here that is more portable.